README.md

```diff
@@ -1 +1,159 @@
-# QuickMaths
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ]  You are using Rspec and Swagger, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍